ccmlib/node.py

```diff
@@ -1491,8 +1498,10 @@ def _find_pid_on_windows(self):
 
     def _get_directories(self):
         dirs = []
-        for i in ['data', 'commitlogs', 'saved_caches', 'logs', 'conf', 'bin', os.path.join('data', 'hints')]:
+        for i in ['commitlogs', 'saved_caches', 'logs', 'conf', 'bin', 'hints']:
```

Comment:
I think so, the default hint directory in 3.0+ is actually in $CASSANDRA_HOME/data/hints and the saved_caches directory default to $CASSANDRA_HOME/data/saved_caches etc